src/components/colors/Colors.module.scss

```diff
@@ -1,16 +1,18 @@
-.colors__color-block__item {
+@import '../../styles/variables/tokens.scss';
```

Comment:
This import shouldn't be necessary since variables are applied to the :root. Commenting out this import has no impact on the component.

Reply:
Right you are. I was still in the sass compile-time mindset.